cards.js

```diff
@@ -16,8 +17,67 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
+function buildDeck()
+{
+  var deck_array = []
+
+  for(var suits_i = 0; suits_i < suits.length; suits_i++)
+  {
+    for(var values_i = 0; values_i < values.length; values_i++)
+    {
+      deck_array.push(
+          {suit: (suits[suits_i]),
+          value: (values[values_i])}
+        );
+    }
+  }
+
+  return deck_array
+}
+
+/*function deal(){
+for(var deck_i = 0; deck_i < deck.length; deck_i++)
+{
+      var dealt = deck.pop;
+      return dealt;
+}
+}
+*/
+
+
+var deck    = shuffle(buildDeck());
+
+var dealtPlayer1 = deck[0]
+var dealtPlayer2 = deck[1]
 
```

Comment:
if i'm being nitpicky about naming, whats being stored in `dealtPlayer1` might be better to have `player1Card` or something to that effect